apache/superset
# fix: Revert "fix(sqllab): flaky json explore modal due to over-rendering (#26156)"

Fixes #26276

### SUMMARY
This reverts commit 77c73b63db8fe9574db129e85b4393fa33f283a6. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:

https://github.com/apache/superset/assets/1392866/46bda7e2-f567-4aef-83b9-35633c6dcdf2

After:

https://github.com/apache/superset/assets/1392866/7d4da06a-f3df-4e81-b7ed-189aba932ef2



### TESTING INSTRUCTIONS
Go to sqllab and set async query enabled
Click on run query that takes more than 5 seconds
See the result content

### ADDITIONAL INFORMATION
